app/models/invoice.rb

```diff
@@ -96,4 +98,14 @@ def calculate_total_amount
   def set_currency
     invoice_items.each{ |i| i.update(currency: currency) }
   end
+
+  def check_bank_account
+    return true if bank_account_id.blank?
```

Comment:
@yunixon it's useless to return `true` or `false` in validation or callback. If you want to prevent saving than you make `throw :abort` (for callbacks) or `errors.add` (for validations). If you don't want to prevent saving, then just return nothing.